### PR TITLE
initial implementation

### PR DESCRIPTION
**Pull request by GiorgosKarantonis**
Finished initial implementation

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -16,8 +16,37 @@
...
+@cli.command(name='push')
+@click.argument('src', nargs=1)
+@click.argument('dst', nargs=1)
+def push(src, dst):
+    '''
+        [SRC] [DST]
+        Push a container image from a source to an external registry
+    '''
+    skopeo_cmd = f'skopeo copy {src} {dst}'
+    run_command(skopeo_cmd)
```
Comment (fridex) [inline app.py]: What happens if anything goes wrong here?

**Comment by goern** [comment]
why do we need this component? pushing images from the openshift internal registry to some other registry is a well known task and implemented by (at least) one of our tekton tasks

**Comment by fridex** [comment]
> why do we need this component? pushing images from the openshift internal registry to some other registry is a well known task and implemented by (at least) one of our tekton tasks

To have an ability to run this as part of an argo workflow.

**Review by fridex — APPROVED** [review]
lgtm, thanks! :+1:

**Pull request merged**